swagger_server/itm/itm_action_handler.py

```diff
@@ -118,6 +118,9 @@ def __successful_treatment(self, treatment: str, injury_name: str, location: str
                 return treatment == SupplyTypeEnum.SPLINT
             case InjuryTypeEnum.CHEST_COLLAPSE:
                 return treatment == SupplyTypeEnum.DECOMPRESSION_NEEDLE
+            case InjuryTypeEnum.ABRASION:
+                if 'face' in location:
```

Comment:
Sorry, I forgot to add to the ticket the locations where they want the abrasion:
- left face, right face, right thigh, left thigh, right calf, left calf, right bicep, left bicep, right forearm, left forearm

Reply:
Is the treatment for all abrasion locations a `pressure bandage`?

Reply:
> Is the treatment for all abrasion locations a `pressure bandage`?

Yes.

Reply:
I was told back abrasion too, I thought